tests/unit/views/Runs.spec.js

```diff
@@ -149,19 +86,126 @@ describe('Runs.vue', () => {
 
     it('will filter the libraries in the table', () => {
       expect(wrapper.find('tbody').findAll('tr').length).toEqual(1)
-      expect(wrapper.find('tbody').findAll('tr').at(0).text()).toMatch(/TRAC-456/)
+      expect(wrapper.find('tbody').findAll('tr').at(0).text()).toMatch(/TRAC-123/)
+    })
+  })
+
+  describe('start button', () => {
+    let button
+
+    it('is enabled when the run state is pending', () => {
+      // run at(3) is in state pending
+      button = wrapper.find('tbody').findAll('tr').at(3).findAll('td').at(4).find('#startRun')
```

Comment:
would it be worth adding an id to each run within the runs table. This would make testing much easier so we could do e.g wrapper.find('#run1').find('#startRun')?

Reply:
Yep sounds good, will do that

Reply:
Might be a little tricky as the b-table doesnt seem to have anything built in to add id for the row